src/app/content/components/Attribution.spec.tsx

```diff
@@ -177,5 +177,27 @@ describe('Attribution', () => {
 
       expect(details.children[1].innerHTML).toMatch(`Authors: Jhon Doe, Jonny Doe`);
     });
+
+    it('renders special licensing and attribution for HS Physics', async() => {
+
+      const { node } = renderToDom(render());
+      const details = node;
+
+      if (!document) {
+        return expect(document).toBeTruthy();
+      }
+
+      details.setAttribute('open', '');
+
+      store.dispatch(
+        actions.receiveBook({...formatBookData(book, mockCmsBook), id: 'cce64fde-f448-43b8-ae88-27705cceb0da'})
+      );
+
+      expect(details.children[1].innerHTML).toMatch(`you must attribute “Texas Education Agency (TEA)."`);
```

Comment:
Maybe this could check for a match with `teaText` from `attributionText.ts`? I guess there might be some errors with comparing a `string` with `innerHTML` which is already parsed, but maybe this can be solved by using `DOMParser` to parse also `teaText` before comparing them?

Reply:
Actually it gets complicated because of the variables needed, I would have to pass in every single variable for DomParser to work, that's why it's easier to just check if TEA is included in the text.